vscode-dotnet-runtime-library/src/Acquisition/DotnetCoreAcquisitionWorker.ts

```diff
@@ -532,7 +532,7 @@ ${WinMacGlobalInstaller.InterpretExitCode(installerResult)}`), install);
         dotnetPath = await installer.getExpectedGlobalSDKPath(installingVersion,
             context.acquisitionContext.architecture ?? this.getDefaultInternalArchitecture(context.acquisitionContext.architecture));
 
-        context.installationValidator.validateDotnetInstall(install, dotnetPath, os.platform() !== 'win32', os.platform() !== 'darwin');
+        context.installationValidator.validateDotnetInstall(install, dotnetPath, os.platform() === 'darwin', os.platform() !== 'darwin');
```

Comment:
Was looking at what this method does, confused by one of the errors this returns: `Expected executable file exists but is not a file` I'm guessing this is along the lines of you expect there to be a dotnet executable file, but instead you have a folder named dotnet

Reply:
That is correct -- it also checks whether the dotnet folder exists and has an executable in it, though in this case we just check that the executable is there and a file instead of a directory called dotnet